### ISSUE TITLE
[Feature Request]: Support log level override for multi-language pipeline

### ISSUE THREAD

**Issue by Abacn** [apache/beam#24647]
### What would you like to happen?

Users using Python multi-language ReadFromKafka transform observes excessive INFO level log spam emitted from logger `org.apache.kafka.clients.consumer.internals.SubscriptionState`:
```
message: "[Consumer clientId=consumer-Reader-27_offset_consumer_168302404_mz-insights-dataflow-v1.27-42,
groupId=Reader-27_offset_consumer_168302404_mz-insights-dataflow-v1.27] Seeking to LATEST offset of partition
***.v2-27"
```
However, currently log level override pipeline option (supported in Java and Python SDK) does not take effect for multi-language pipeline.
We already done some similar tasks (#23635) where Python pipeline running on flink runner (through flink job-server Java artifacts) supports work level overrides for java loggers. We should be able to add support to java container also.

### Issue Priority

Priority: 3 (nice-to-have improvement)

### Issue Components

- [X] Component: Python SDK
- [X] Component: Java SDK
- [ ] Component: Go SDK
- [ ] Component: Typescript SDK
- [ ] Component: IO connector
- [ ] Component: Beam examples
- [ ] Component: Beam playground
- [ ] Component: Beam katas
- [ ] Component: Website
- [ ] Component: Spark Runner
- [ ] Component: Flink Runner
- [ ] Component: Samza Runner
- [ ] Component: Twister2 Runner
- [ ] Component: Hazelcast Jet Runner
- [ ] Component: Google Cloud Dataflow Runner

**Comment by chamikaramj** [comment]
You can use the following workaround to control Java log level from a Python Multi-language pipeline.

https://cwiki.apache.org/confluence/display/BEAM/Multi-language+Pipelines+Tips#MultilanguagePipelinesTips-HowtosetJavaloglevelfromaPythonpipelinethatusesJavatransforms

**Comment by Abacn** [comment]
After testing I find the existing Python pipeline option `--sdk_harness_log_level_overrides '{\"org.apache.kafka\":\"ERROR\"}'` also works. This is after #22077 done. Will update the wiki and we can close this issue for now
